index.html

```diff
@@ -245,35 +269,57 @@ <h1>About Me</h1>
-                
-                    <input type="email" name="email" placeholder="Enter address" required>
-            
+                   <div class="form-style">
+                       <div class="input">
+                    <input type="text" maxlength="30" placeholder="First Name" required>
+                    <input type="text" maxlength="30" placeholder="Last Name" required>
+                </div>
+                <div class="em">
+                    <input type="text" name="email" placeholder="Enter address" required >
+                </div>
                 
                     <textarea class="text" name="msg" maxlength="500" placeholder="Enter text here" required></textarea>
+                </div>
                 <div class="button-cont">
                     <button type="submit" class="potfolio-but">Get in Touch</button>
                 </div>
             </form>
-                <div class="flex-container">
-                    <div><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></div>
-                    <div><a href="https://www.linkedin.com/in/richard-friday-54980718a"><img src="./images/Linkedin.png" alt="linkedin"></a></div>
-                    <div><a><img src="./images/medium.png" alt="medium"></a></div>
-                    <div><a href="https://github.com/Richez003"><img src="./images/github.png" alt="github"></a></div>
-                    <div><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></div>
-                    </div>
-                    <a class="last-image" href="#"><img src="./images/illustration-contact-form.png" alt="illustrator"></a>     
-            </section>
-            <div>
-            <p class="lp">Richemonday@gmail.com</p>
-            </div>              
+            <div class="container-flex">
+                <div><a href="https://twitter.com/richardfriday14"><img src="./images/twitter.png" alt="twitter"></a></div>
+                <div><a href="https://www.linkedin.com/in/richard-friday-54980718a"><img src="./images/Linkedin.png" alt="linkedin"></a></div>
+                <div><a><img src="./images/medium.png" alt="medium"></a></div>
+                <div><a href="https://github.com/Richez003"><img src="./images/github.png" alt="github"></a></div>
+                <div><a href="https://angel.co/jobs?ref=onboarding"><img src="./images/angellist.png" alt="angellist"></a></div>
+          </div>
+                <a class="last-image" href="#"><img src="./images/illustration-contact-form.png" alt="illustrator"></a>     
+        </section>
+        <div>
+        <p class="lp">Richemonday@gmail.com</p>
+        </div>              
+            <div>  
+                    <a class="image5" href="#"><img src="./images/illustration-contact-form.png" alt="illustrator"></a>    
+                <!-- <a href="" > <img src="./images/Frame@2x.png" alt="" class="last-frame"></a> -->
```

Comment:
- [ ] To make your code clean...
Please, clear your project from unused commented codes!

Reply:
done

Reply:
done